index.rst

```diff
@@ -302,107 +305,152 @@ Is this case, we could have something like:
+        - No label shall exist having the name ``default``
+
+#.  If a CSC receives a ``start`` command with an empty ``configuration`` (see :ref:`renaming proposal <section-renaming>`) attribute, it shall load the values in ``_init.yaml`` then the site-specific file (e.g. ``_summit.yaml``).
+#.  If a CSC receives a ``start`` command with a ``configuration`` attribute equal to a label in ``_labels.yaml``, it shall load the values in ``_init.yaml``, then then the site-specific file (e.g. ``_summit.yaml``), and lastly the override file.
 
-#.  If a CSC receives a ``start`` command with an empty ``configuration`` (see :ref:`renaming proposal <section-renaming>`) attribute, it shall load the values in ``_base.yaml``.
-#.  If a CSC receives a ``start`` command with a ``configuration`` attribute equal to a label in ``_labels.yaml``, it shall load the values in ``_base.yaml`` first and override those values defined in the mapped configuration file.
-#.  The name ``default`` shall not be used for labels.
 #.  All valid configurations shall have a label to be loaded by the CSC.
-    Configurations cannot be loaded by filename.
-#.  The configuration repository should be kept clean of configurations used for unit testing or for different purposes (e.g. test stand configurations).
-    Configurations tailored for test stand can be kept in different branches/tags.
+
+#.  The configuration repository shall not contain configurations used for unit testing.
+    Override configurations that are site specific should contain the site name as a prefix to the filename (e.g. ``summit_simple_algorithm.yaml``).
     Configurations needed for unit testing shall be added to the ``test`` directory in the CSC repository and use the override feature in CSCs (see `Salobj documentation <https://ts-salobj.lsst.io>`__).
-#.  All configuration files shall have a header explaining that they are loading basic values from ``_base.yaml``, as shown in the :download:`example configuration file <_static/ATSpectrograph_example_config.yaml>` mentioned above.
+#.  All configuration files shall have a header metadata fields explaining that they are loading basic values from ``_init.yaml``, as shown in the :download:`example configuration file <_static/ATSpectrograph_example_config.yaml>` mentioned above.
+
+
+Required Unit and Continuous Integration (CI) Testing
+-----------------------------------------------------
+
+Due to the dependence of the configuration files on the defined schema, which are located in different repositories, CI tests are required to ensure there is no breakage when making modification in either repository.
+
+The following CI tests are required on all configuration repos (e.g. ``ts_config_attcs``):
+
+    #. Verify that location configuration files exist for all sites
+    #. Verification that ``_init.yaml`` plus each of the site-dependent configuration files provide a full and valid schema
```

Comment:
This list and the next one are an awkward mix of "Verify" and "Verification". I suggest using "Verify" for every entry -- use the active voice! This may require a few trivial tweaks to the entries, for example "provide" -> "provides" in this entry.

Reply:
I find "plus" a bit confusing here. Please consider something like "Verify that each site-specific file, together with `_init.yaml`, provides a full and valid schema."

Reply:
Yes, "Validation" is actually the wrong word. Validation means ensuring you built the correct thing, not just created a file that can be read"